mkosi

```diff
@@ -2909,6 +2909,64 @@ def setup_package_cache(args: CommandLineArguments) -> Optional[tempfile.Tempora
     return d
 
 
+class ArgumentConfigParser():
+    options = dict()
+    argparser = None
+    curr_group_title = None
+    curr_group = None
+    FLAGS = enum.Enum('FLAGS', 'Extend')
```

Comment:
I don't understand why those class variables are need. At least `argparser` is unconditionally hidden by the assignment in `__init__`.

Reply:
Just initialize the defaults. I could turn them into object members and initialize them in `__init__()`, but here it has the same effect.